src/notation/internal/notationuiactions.cpp

```diff
@@ -1685,68 +1685,68 @@ const UiActionList NotationUiActions::m_actions = {
-             TranslatableString("action", "Advance whole note (F.B./Chord symbol)"),
-             TranslatableString("action", "Advance cursor: whole note (figured bass/chord symbols)")
+             TranslatableString("action", "Advance cursor: whole note (chord symbols/figured bass)"),
+             TranslatableString("action", "Advance cursor: whole note (chord symbols/figured bass)")
              ),
     UiAction("advance-2",
              mu::context::UiCtxProjectFocused,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Advance half note (F.B./Chord symbol)"),
-             TranslatableString("action", "Advance cursor: half note (figured bass/chord symbols)")
+             TranslatableString("action", "Advance cursor: half note (chord symbols/figured bass)"),
+             TranslatableString("action", "Advance cursor: half note (chord symbols/figured bass)")
              ),
     UiAction("advance-4",
              mu::context::UiCtxProjectFocused,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Advance quarter note (F.B./Chord symbol)"),
-             TranslatableString("action", "Advance cursor: quarter note (figured bass/chord symbols)")
+             TranslatableString("action", "Advance cursor: quarter note (chord symbols/figured bass)"),
+             TranslatableString("action", "Advance cursor: quarter note (chord symbols/figured bass)")
              ),
     UiAction("advance-8",
              mu::context::UiCtxProjectFocused,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Advance eighth note (F.B./Chord symbol)"),
-             TranslatableString("action", "Advance cursor: eighth note (figured bass/chord symbols)")
+             TranslatableString("action", "Advance cursor: eighth note (chord symbols/figured bass)"),
+             TranslatableString("action", "Advance cursor: eighth note (chord symbols/figured bass)")
              ),
     UiAction("advance-16",
              mu::context::UiCtxProjectFocused,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Advance 16th note (F.B./Chord symbol)"),
-             TranslatableString("action", "Advance cursor: 16th note (figured bass/chord symbols)")
+             TranslatableString("action", "Advance cursor: 16th note (chord symbols/figured bass)"),
+             TranslatableString("action", "Advance cursor: 16th note (chord symbols/figured bass)")
              ),
     UiAction("advance-32",
              mu::context::UiCtxProjectFocused,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Advance 32nd note (F.B./Chord symbol)"),
+             TranslatableString("action", "Advance cursor: 32nd note (figured bass/chord symbols)"),
              TranslatableString("action", "Advance cursor: 32nd note (figured bass/chord symbols)")
```

Comment:
Oh wait, this one still needs the chord symbols/figured bass swap

Reply:
Yep, later, after choir rehearsal 😉

Reply:
Done